sql/catalyst/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala

```diff
@@ -2322,6 +2328,8 @@ class SQLConf extends Serializable with Logging {
 
   def defaultV2Catalog: Option[String] = getConf(DEFAULT_V2_CATALOG)
 
+  def legacyCTESubstitutionEnabled: Boolean = getConf(LEGACY_CTE_SUBSTITUTION_ENABLED)
```

Comment:
nit: since this is used only once, shall we avoid the definition of this method and just use `getConf` of the property where needed?

Reply:
fixed